Code/src/main/java/nl/utwente/group10/ui/components/InputAnchor.java

```diff
@@ -57,4 +57,10 @@ public boolean canConnect() {
         return !isConnected();
     }
 
+
+
+	@Override
+	public String toString() {
+		return "InputAnchor for " + getBlock();
```

Comment:
Deze toString() lijkt niet helemaal op de conventie die in de backend aangehouden wordt, maar dat is niet zo'n probleem.